torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -375,7 +375,10 @@ Node* insertEmbeddingBagOps(Node* observer, const std::string& op_name) {
   }
 
   qembedding_bag_inputs.push_back(none); // compressed_indices_mapping
-  qembedding_bag_inputs.push_back(embedding_bag_inputs[inputs_size - 1]);
+  qembedding_bag_inputs.push_back(embedding_bag_inputs[inputs_size - 2]);
+
+  TORCH_CHECK(embedding_bag_inputs[inputs_size - 1]->mustBeNone(),
```

Comment:
@jbschlosser, as far as I could figure out so far, these are the minimum amount of changes I had to make with respect to quantization for this PR. All this does is throw an error if quantization is attempted on an embedding bag with a padding_idx defined. Note, I also added a test in `test/quantization/test_quantize_jit.py` to make sure the error is raised correctly

Reply:
Awesome, thanks for taking the time to get this straightened out! Will attempt the merge again now.